### PR TITLE
Translation: JavaScript Fundamentals > Type Conversions

### PR DESCRIPTION
**Pull request by Zim123**

### CODE REVIEW

File: 1-js/02-first-steps/06-type-conversions/article.md [hunk 1]
```diff
@@ -1,160 +1,160 @@
...
-For example, `alert` automatically converts any value to a string to show it. Mathematical operations convert values to numbers.
+Наприклад, `alert` автоматично перетворює будь-яке значення в рядок, щоб показати його. Математичні операції перетворюють значення на числа.
 
-There are also cases when we need to explicitly convert a value to the expected type.
+Є також випадки, коли нам необхідно явно перетворити значення на очікуваний тип.
 
-```smart header="Not talking about objects yet"
-In this chapter, we won't cover objects. Instead, we'll study primitives first. Later, after we learn about objects, we'll see how object conversion works in the chapter <info:object-toprimitive>.
+```smart header="Поки що не говоримо про об'єкти"
+У цій главі ми не будемо охоплювати об'єкти. Замість цього спочатку ми вивчемо примітиви. Пізніше, після того, як ми дізнаємося про об'єкти, ми побачимо, як перетворення об'єктів працює в цьому розділі <info:object-toprimitive>.
```
Comment (tarasyyyk) [inline 1-js/02-first-steps/06-type-conversions/article.md]: Please, follow [Glossary](https://github.com/javascript-tutorial/uk.javascript.info/issues/8) for `chapter` translation.

**Comment by tarasyyyk** [comment]
@Zim123 very good. I made small corrections to your translation. Meged.

P.S. I found out you type "`і`" instead of "`и`". Did you have keyboard layout where English "B" key have corresponding ukrainian "`і`" key (not "`и`" as ussual in Windows)?
I have keyboard layout files (with B => и, S => і), if you'r using macOS

File: 1-js/02-first-steps/06-type-conversions/article.md (hunk 1 above)
Comment (Zim123) [reply 1-js/02-first-steps/06-type-conversions/article.md]: Yes, my bad.

**Comment by Zim123** [comment]
Hm, I'm not sure, maybe some kind of typo, I'm using macOS keyboard as well (with B => и, S => і)
Will be more careful next time.

**Pull request merged**